packages/core/src/components/time-picker-rework/time-picker-rework.scss

```diff
@@ -0,0 +1,119 @@
+/*
+ * SPDX-FileCopyrightText: 2023 Siemens AG
+ *
+ * SPDX-License-Identifier: MIT
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+@import 'common-variables';
+@import 'mixins/shadow-dom/component';
+@import 'components/form/input';
+
+:host {
```

Comment:
The component should have `inline-block` or `block` as display value. Otherwise you cannot control the sizing of the component. Which makes the usages strange like the old time-picker.

Reply:
> The component should have `inline-block` or `block` as display value. Otherwise you cannot control the sizing of the component. Which makes the usages strange like the old time-picker.

resolved, i added display: block;